ably/rest/auth.py

```diff
@@ -154,7 +157,7 @@ def request_token(self, key_name=None, key_secret=None, query_time=None,
         return TokenDetails.from_dict(response_json)
 
     def create_token_request(self, key_name=None, key_secret=None,
-                             query_time=False, token_params=None):
+                             query_time=None, token_params=None):
```

Comment:
The problem I see here is that, say we only want to set the querytime, creating an instance of `AuthOptions`, set it's querytime and passing it to the method seems very un-pythonic.